source/core/ScriptDomain.cpp

```diff
@@ -424,7 +425,23 @@ namespace GTA
 			}
 		}
 
-		Log("[INFO]", "Starting ", _scriptTypes->Count.ToString(), " script(s) ...");
+		//Initialize Console-Script, propably should be moved somewhere else
+		_console = gcnew Console();
+		_console->Script = gcnew ConsoleScript();
+		_console->Script->_running = true;
+		_console->Script->_filename = "internal";
+		_console->Script->_scriptdomain = this;
+
+		_console->Script->_thread = gcnew Thread(gcnew ThreadStart(_console->Script, &Script::MainLoop));
+		_console->Script->_thread->Start();
+		_runningScripts->Add(_console->Script);
```

Comment:
I dont think this should be part of running scripts. Pressing insert shouldnt clear the whole console